pkg/accessscope/validate.go

```diff
@@ -28,6 +28,9 @@ func ValidateSimpleAccessScopeProto(scope *storage.SimpleAccessScope) error {
 // represents valid simple access scope rule.
 func ValidateSimpleAccessScopeRules(rules *storage.SimpleAccessScope_Rules) error {
 	var validationErrs *multierror.Error
+	if rules == nil {
+		validationErrs = multierror.Append(validationErrs, errox.InvalidArgs.New("rules must be set"))
```

Comment:
super nit: seems like we use the term "field" here in the error messages (see line 37), maybe we should use that here as well:
```suggestion
		validationErrs = multierror.Append(validationErrs, errox.InvalidArgs.New("rules field must be set"))
```